microsoft-365/compliance/customer-key-manage.md

```diff
@@ -132,7 +132,7 @@ To create a DEP, you need to remotely connect to SharePoint Online by using Wind
   
 1. On your local computer, using a work or school account that has global administrator permissions in your organization, [Connect to SharePoint Online PowerShell](/powershell/sharepoint/sharepoint-online/connect-sharepoint-online?preserve-view=true&view=sharepoint-ps).
 
-2. In the Microsoft SharePoint Online Management Shell, run the Register-SPODataEncryptionPolicy cmdlet as follows:
+2. In the Microsoft SharePoint Online Management Shell, for a premium/standard key vault run the Register-SPODataEncryptionPolicy cmdlet as follows:
```

Comment:
```suggestion
2. In the Microsoft SharePoint Online Management Shell, for either a Premium or a Standard Key Vault, run the Register-SPODataEncryptionPolicy cmdlet as follows:
```

Reply:
Premium and Standard don't need to be capped unless they're part of the entire brand name. Perhaps write the entire brand out?

Reply:
Thanks, @KCCross, I've changed my suggestion accordingly.